internal/metrics/metrics.go

```diff
@@ -0,0 +1,72 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+	"runtime"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/collectors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.uber.org/zap"
+
+	"k8sgateway/internal/version"
+)
+
+const Namespace = "twingate_gateway"
+
+type Config struct {
+	Port     string
+	Logger   *zap.SugaredLogger
+	Registry *prometheus.Registry
```

Comment:
We can't make these variables private because it's exported and used [here](https://github.com/Twingate/kubernetes-access-gateway/pull/31/files#diff-e3ad76770e5ac8ed654e8c5585782469e67e0d981adb9153604109263f22e93eR86-R88)